template/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -636,6 +225,8 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 13B07F931A680F5B00A75B9A /* Build configuration list for PBXNativeTarget "HelloWorld" */;
 			buildPhases = (
+				FD10A7F022414F080027D42C /* Start Packager */,
```

Comment:
Right here and in the other targets. By default cocoapods should be the first phase right after the `target dependencies`.